src/main/java/org/folio/services/fund/FundUpdateLogService.java

```diff
@@ -41,4 +43,10 @@ public Future<Void> updateFundUpdateLog(FundUpdateLog fundUpdateLog, RequestCont
   public Future<Void> deleteFundUpdateLog(String jobId, RequestContext requestContext) {
     return restClient.delete(resourceByIdPath(FUND_UPDATE_LOGS, jobId), requestContext);
   }
+
+  public Future<SequenceNumber> getJobNumber(RequestContext requestContext) {
+    var requestEntry = new RequestEntry(resourcesPath(JOB_NUMBER))
+      .withQueryParameter("type", "Logs");
```

Comment:
lets be more specific, because different types of logs are possible.
Here is an option:
FUND_BULK_UPDATES_LOGS

Reply:
good idea, thanks, using more simpler FUND_UPDATE_LOGS name